src/mixins/ViewsMixin.js

```diff
@@ -48,7 +54,14 @@ export default {
 			cancelFetchFullForm: () => {},
 
 			// markdown renderer for descriptions
-			markdownit: new MarkdownIt({ breaks: true }),
+			markdownit: new MarkdownIt({ breaks: true })
+				// Add .use to support the mdit plugins
+				.use(align)
+				.use(imgSize)
+				.use(obsidianImgSize)
+				.use(mark)
+				.use(sub)
+				.use(sup)
```

Comment:
```suggestion
				.use(sup),
```
Comma missing to satisfy prettier